.github/scripts/test-no-error-reports.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+RUNDIR="run"
+CRASH="crash-reports"
+SERVERLOG="server.log"
+
+if [[ -d $RUNDIR/$CRASH ]]; then
+  echo "Crash reports detected:"
+  cat $RUNDIR/$CRASH/crash*.txt
+  exit 1
+fi
```

Comment:
You need `nullglob` option to avoid returning the pattern itself when not match exist.
Here a better implementation to get the crash report:
```sh
# enable nullglob to get 0 results when no match rather than the pattern
shopt -s nullglob
# store matches in array (don't forget to double-quote variables expansion
crash_reports=( "$RUNDIR/$CRASH/crash"*.txt )
if [ "${#crash_reports[]}" -gt 0 ]; then
  latest_crash_report="${crash_reports[-1]}"
  {
    printf 'Latest crash report detected %s:\n' "${latest_crash_report##*/}"
    cat "$latest_crash_report"
  } >&2
  exit 1
fi

Reply:
these files come from ExampleMod and should be fixed there